src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24414.cs

```diff
@@ -16,8 +16,34 @@ public Issue24414(TestDevice device)
 		[Category(UITestCategories.Visual)]
 		public void Issue24414Test()
 		{
-			App.WaitForElement("WaitForStubControl");
-			VerifyScreenshot();
+			var label = App.WaitForElement("TheLabel");
+			var ex = TryVerifyScreenshot();
+
+			for (int i = 1; i <= 3; i++)
+			{
+				label.Tap();
+				var exception = TryVerifyScreenshot("Issue24414Test_" + i);
+				ex ??= exception;
+			}
+
+			if (ex != null)
+			{
+				throw ex;
+			}
+		}
+
+		private Exception? TryVerifyScreenshot(string fileName)
```

Comment:
The build is failing at this point, could you take a look?
`D:\a\_work\1\s\src\Controls\tests\TestCases.Shared.Tests\Tests\Issues\Issue24414.cs(20,13): error CS7036: There is no argument given that corresponds to the required parameter 'fileName' of 'Issue24414.TryVerifyScreenshot(string)' [D:\a\_work\1\s\src\Controls\tests\TestCases.iOS.Tests\Controls.TestCases.iOS.Tests.csproj]`

Reply:
@jsuarezruiz stupid mistake, I'm sorry.
It's fixed now, you can run AZP again, thanks!